types/test/data-types.ts

```diff
@@ -32,3 +32,11 @@ expectTypeOf(INTEGER()).toEqualTypeOf<DataTypes.IntegerDataType>();
 expectTypeOf(new INTEGER()).toEqualTypeOf<DataTypes.IntegerDataType>();
 expectTypeOf(INTEGER.UNSIGNED.ZEROFILL()).toEqualTypeOf<DataTypes.IntegerDataType>();
 expectTypeOf(new INTEGER.UNSIGNED.ZEROFILL()).toEqualTypeOf<DataTypes.IntegerDataType>();
+
+// JSON
+expectTypeOf(new JSON()).toEqualTypeOf<DataTypes.AbstractDataType>();
+expectTypeOf(JSON()).toEqualTypeOf<DataTypes.AbstractDataType>();
+
+// JSONB
+expectTypeOf(new JSONB()).toEqualTypeOf<DataTypes.AbstractDataType>();
+expectTypeOf(JSONB()).toEqualTypeOf<DataTypes.AbstractDataType>();
```

Comment:
Can you make sure that there is an empty line at the end of this file?

Reply:
Oops, apparently I failed at the new-line even after review. Looking into it... I have the newline locally, but it seems not to be showing up on github, which is confusing me. Looking into it....